test/util/velero/install.go

```diff
@@ -276,10 +265,10 @@ func installVeleroServer(ctx context.Context, cli, cloudProvider string, options
 		}
 	}
 
-	if len(options.Features) > 0 {
-		args = append(args, "--features", options.Features)
-		if strings.EqualFold(options.Features, FeatureCSI) && options.UseVolumeSnapshots {
-			if strings.EqualFold(cloudProvider, "azure") {
+	if strings.EqualFold(cloudProvider, "azure") {
```

Comment:
```suggestion
	if len(options.Features) > 0 {
		args = append(args, "--features", options.Features)
		if strings.EqualFold(options.Features, FeatureCSI) && options.UseVolumeSnapshots &&
		      strings.EqualFold(cloudProvider, "azure")  {
			fmt.Println("Start to install Azure VolumeSnapshotClass ...")
			if err := KubectlApplyByFile(ctx, "../util/csi/AzureVolumeSnapshotClass.yaml"); err != nil {
				return err
```

If options.Features is not empty, arg --features must be set with that value no matter what provider it is.

Reply:
Current code blocked data mover pipeline due to args `--features` is missing.

Reply:
ack